jwst/resample/tests/test_resample_step.py

```diff
@@ -144,10 +149,26 @@ def miri_rate():
         'readpatt': 'FAST',
         'sample_time': 10.0,
         'start_time': 58119.8333,
-        'type': 'MIR_LRS-SLITLESS',
+        'type': 'MIR_LRS-FIXEDSLIT',
         'zero_frame': False}
 
-    return im
+    yield im
+    im.close()
+
+
+@pytest.fixture
+def miri_rate_pair(miri_rate_zero_crossing):
+    im1 = miri_rate_zero_crossing
+    # Create a nodded version
+    im2 = im1.copy()
+    im2.meta.wcsinfo.ra_ref = 0.00026308279776455
```

Comment:
since these constants are re-used above, could they be named something like RA_OFFSET and DEC_OFFSET and put at the top of the file?

Reply:
The values are actually a little different, on purpose, so I don't think making them named constants gains much.